src/components/Login.js

```diff
@@ -82,6 +82,13 @@ export default class Login extends Component {
           <button className="buttons register-button" onClick={this.register}>
             Register
           </button>
+          <p>
+            Email us :
```

Comment:
In the "Sign in to report your sightings!" header, we could change it to be "Sign in or email to report your sightings!" and the email part could be clicked on to send an email? @AntonioDaria

Reply:
Have a look at the adjustments